File: pdflib-3.0.2/bind/java/hello.java
/* hello.java
 * Copyright (C) 1997-2000 Thomas Merz. All rights reserved.
 *
 * PDFlib client: hello example in Java
 */

import java.io.*;
import com.pdflib.pdflib;

public class hello
{
    public static void main (String argv[]) throws
	OutOfMemoryError, IOException, IllegalArgumentException,
	IndexOutOfBoundsException, ClassCastException, ArithmeticException,
	RuntimeException, InternalError, UnknownError
    {
	int font;
	pdflib p;

	p = new pdflib();

	if (p.open_file("hello_java.pdf") == -1) {
	    System.err.println("Couldn't open PDF file hello_java.pdf\n");
	    System.exit(1);
	}

	p.set_info("Creator", "hello.java");
	p.set_info("Author", "Thomas Merz");
	p.set_info("Title", "Hello world (Java)");

	p.begin_page(595, 842);

	font = p.findfont("Helvetica-Bold", "host", 0);

	p.setfont(font, 18);

	p.set_text_pos(50, 700);
	p.show("Hello world!");
	p.continue_text("(says Java)");
	p.end_page();

	p.close();
    }
}

File: pdflib-3.0.2/bind/perl/hello.pl
#!/usr/bin/perl
# hello.pl
# Copyright (C) 1997-2000 Thomas Merz. All rights reserved.
#
# PDFlib client: hello example in Perl
#

use pdflib_pl 3.02;

$p = PDF_new();

die "Couldn't open PDF file" if (PDF_open_file($p, "hello_pl.pdf") == -1);

PDF_set_info($p, "Creator", "hello.pl");
PDF_set_info($p, "Author", "Thomas Merz");
PDF_set_info($p, "Title", "Hello world (Perl)");

PDF_begin_page($p, 595, 842);
$font = PDF_findfont($p, "Helvetica-Bold", "host", 0);

PDF_setfont($p, $font, 18.0);

PDF_set_text_pos($p, 50, 700);
PDF_show($p, "Hello world!");
PDF_continue_text($p, "(says Perl)");

PDF_end_page($p);
PDF_close($p);

PDF_delete($p);
